api/src/main/java/org/pmiops/workbench/config/StoredCredentialsConfig.java

```diff
@@ -0,0 +1,48 @@
+package org.pmiops.workbench.config;
+
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import java.io.IOException;
+import org.pmiops.workbench.auth.Constants;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.pmiops.workbench.google.CloudStorageServiceImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Lazy;
+
+@Configuration
+@Import(CloudStorageServiceImpl.class)
+public class StoredCredentialsConfig {
+  /**
+   * Service account credentials for Gsuite administration, corresponding to the "gsuite-admin"
+   * service account in each environment. Enabled for domain-wide delegation of authority.
+   */
+  @Lazy
+  @Bean(name = Constants.GSUITE_ADMIN_CREDS)
```

Comment:
nit: After fiddling with this, I remember why this has to be a constant string rather than an enum name. Still, I'd likely rename `Constants` to `AuthBeanNames`.